tests/unit/test_getnid.py

```diff
@@ -68,9 +68,17 @@ def loadfile(self, relativepath, filename):
         path = current.split('/')
         finalpath = ''
         finalstring = ''
+        texttofind = ''
 
         try:
-            index_to_glancesync = path.index('fiware-glancesync') + 1
+            if 'fiware-glancesync' in path:
+                texttofind = 'fiware-glancesync'
+            elif 'workspace' in path and 'develenv' in path:
+                texttofind = 'workspace'
```

Comment:
I do not like too much this solution because we are doing the code depending on the way that users perform the 'clone' of the repo. What if users clone the repo into a folder with a different name?

Could we work with relatives paths instead of absolute ones? Unit tests should be always executed from the main 'folder' of the repo.